addons/l10n_it/migrations/15.0.0.3/post-migrate.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from odoo import api, SUPERUSER_ID
+
+def migrate(cr, version):
+
+    cr.execute("""
+        INSERT INTO account_account_account_tag
+        SELECT a.id, att.account_account_tag_id
+        FROM account_account AS a
+        JOIN account_account_template AS at ON a.code LIKE CONCAT(at.code, '%')
```

Comment:
Please use longer names for your aliases XD
Names with only a few letters are usually a very bad idea: at and att are nearly the same name ; a isn't that different. Overall, it makes a simple query more confused. 

My suggestion:
account_account ==> account
account_account_template ==> template 
account_account_template_account_tag ==> template_tag

imd for ir_model_data is fine, as there is nothing with close name here.